js/index.js

```diff
@@ -0,0 +1,51 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector('footer');
+const copyRight = document.createElement('p');
+
+copyRight.innerHTML = `&copy Raissa Gomes ${thisYear}`;
+footer.appendChild(copyRight);
+
+const skills = ['HTML' 'CSS' 'JavaScript'];
+const skillsSection = document.getElementById('skills');
+const skillsList = skillsSection.querySelector('ul')
+
+for (let i = 0; i < skills.length; i++) {
+    const skill = document.createElement('li');
+    skill.innerHTML = skills[i];
+    skillsList.appendChild(skill);
+}
+
+const messageForm = document.getElementsByName('leave_message');
+const messageSection = document.getElementById('messages');
+messageSection.hidden = true;
```

Comment:
Great job with the optional task here! If you want another challenge, how would you handle the fact that if people use the "Remove" button to remove all their messages, we want to hide the message section again?

**Hints**: it would happen in the `removeButton.addEventListener` part of the code; and you can use the [Element.children](https://developer.mozilla.org/en-US/docs/Web/API/Element/children) method to figure out if there's any messages in the `<ul>` tag in that section.